index.js

```diff
@@ -91,7 +92,10 @@ function compile(entry) {
         safe: false, // load .env.example (defaults to "false" which does not use dotenv-safe)
         silent: true
       }),
-      new RemoveSourceMapUrlWebpackPlugin({})
+      new RemoveSourceMapUrlWebpackPlugin({}),
+      new webpack.DefinePlugin({
+        'process.env.REACT_APP_VERSION': JSON.stringify(process.env.REACT_APP_VERSION),
```

Comment:
Yeah it adds quotes. Otherwise the replacement jams the raw number in, so you end up with invalid javascript:

```
export const getAppVersion = (): Option<string> => 1.0.37735
```

`JSON.stringify` was the pattern used in their docs (e.g. `VERSION: JSON.stringify('5fa3b9'),`) so I just went with it.

Reply:
oh wow! Well, that's the API 😁